### DISCUSSION TITLE
create view inside a folder

### DISCUSSION THREAD

**Discussion by sanjai-mahalingan** [FilledStacks/stacked#1044] [Q&A]
Hi, is it possible to create a view inside a folder, if yes, can you please guide me..

**Comment by FilledStacks** [comment]
I'm assume you're talking about creating a view using the CLI?

**Reply by sanjai-mahalingan** [reply]
When we do "stacked create view login" stack will create a view and viewmodel inside views,
but i want to create a login view & viewmodel inside a new folder (auth) inside a views folder.

eg: 
lib/ui/views/auth/login
-> login_view.dart
-> login_viewmodel.dart

lib/ui/views/auth/signup
-> signup_view.dart
-> signup_viewmodel.dart

**Comment by FilledStacks** [comment]
You can use the `-c` option to pass a path where you want to create the view or a service.
